nginx/nginx.conf

```diff
@@ -106,9 +106,10 @@ http {
         location /tuesday { return 302 https://pledge.wnyc.org/support/radiolab; }
 
         # Podcast Membership
-        location ~ ^/donate/? { return 302 https://pledge.wnyc.org/support/radiolab?utm_medium=audio&utm_source=podcast-spot&utm_campaign=rl-cye-addgift; }
+        location ~ ^/donate/? { return 302 https://pledge.wnyc.org/support/radiolab?utm_medium=audio&utm_source=redirect&utm_campaign=radiolab-dot-org-slash-donate ; }
         location ~ ^/(insider|20|twenty)/?$ { return 301 $not_found; }
-        location ~ ^/(join|membership|members|member)/?$ { return 301 $members; }
+        location ~ ^/join/? { return 302 https://members.radiolab.org?utm_source=audio&utm_medium=redirect&utm_campaign=radiolab-dot-org-slash-join; }
+        location ~ ^/(membership|members|member)/?$ { return 302 $members; }
```

Comment:
Man, this is tough... The change from 301 to 302 won't work, will it?

Reply:
Unfortunately these are the cards we are dealt.